docs/_tutorials/read-only-json-api.md

```diff
@@ -0,0 +1,321 @@
+---
+title: Read-Only JSON API
+author: izdwuut
+date: 2020-12-23 10:04:00 +0100
+---
+
+There shouldn't be a necessity to explain what is the [API](https://en.wikipedia.org/wiki/API). I assume that you've landed here to learn it in a context of Jekyll. Let's jump straight to the actual thing, shall we?
```

Comment:
I think the tone and the info here can be improved to not presume to much and also to tell the reader in the first line what the article is going to create and why they should care.

e.g.

```
Here we take your posts markdown files in your Jekyll site and make them available as static JSON files on a kind of REST API. Then you can view them directly in the browser as `/foo.json` and you can use them in your frontend layer, such as if you have a JavaScript application on the frontend which pulls in JSON content.

We are using [JSON](https://en.wikipedia.org/wiki/JSON) is a popular cross-language method of transferring data on the web, without all the presentation and styling of HTML. The files we output will be available in a similar way to if you built a [REST](https://restfulapi.net/) [API](https://en.wikipedia.org/wiki/API) in a server-side language like Ruby, Python or Node. 
```

Reply:
Fixed. yeah, I agree that tone is unpleasant. It just that I've been too blunt about caring about the reader time.